src/server/pkg/sync/sync.go

```diff
@@ -38,6 +39,75 @@ func NewPuller() *Puller {
 	}
 }
 
```

Comment:
Like a summary of the whole file? At the top it says:

```go
// Package sync provides utility functions similar to `git pull/push` for PFS
```

that seems like a pretty good high level summary although I could expand on it a little more.